lib/ts-sdk/src/schemas/zod/fields.ts

```diff
@@ -23,7 +23,7 @@ const EventBaseSchema = z.object({
   eventType: EventTypeEnum,
 
   /** Description of what this event represents */
-  description: z.string().optional(),
+  description: z.string().nullish(),
```

Comment:
See my note in the PR explaining this switch. 

**TL;DR** it's necessary to avoid raising a validation error when parsing optional fields that have been set to `null` by pydantic during serialization